nengo_gui/static/components/netgraph.js

```diff
@@ -683,3 +683,16 @@ Nengo.NetGraph.prototype.scaleMiniMapViewBox = function () {
     this.view.setAttribute('width', w / this.scale);
     this.view.setAttribute('height', h / this.scale);
 }
+
+/** Called from the individual component ondragmove handlers in order to pan
+ * the entire view whenever CTRL or the middle mouse button are pressed.
+ * Returns true if the event has been processed by the NetGraph component and
+ * no further event processing must be performed by the calling code.
+ * Returns false otherwise. */
+Nengo.NetGraph.prototype.capture_move_event = function (event) {
+    if (event.ctrlKey || (event.buttons & 4)) {
```

Comment:
Yes, as part of the MouseEvent specification. See https://www.w3.org/TR/uievents/#mouseevent

If you think this makes any sense, I can surely add constants for the mouse buttons instead of relving to rely on magic numbers.